### PR TITLE
#167727717 Integrate TravisCI

### PR DESCRIPTION
**Pull request by jsbuddy**
#### What this PR does
This PR integrates TravisCI and adds build status badge to `readme`

#### Tasks completed
- create `travis.yml` and add configuration
- add status badge to `readme.md`

#### How can this be manually tested
- Clone the repo
- Navigate to `readme.md`

#### What are the relevant pivotal tracker stories?
[#167727717](https://www.pivotaltracker.com/story/show/167727717)

### CODE REVIEW

File: travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+sudo: false
+
+language: node_js
+
+node_js:
+  - "node"
+
+after_success: npm run coverage
```
Comment (ladaposamuel) [inline travis.yml]: You forgot the new line

**Review by ladaposamuel — CHANGES_REQUESTED** [review]

**Comment by tejiri4** [comment]
You test is not running try placing the right command and install dependency

File: travis.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+sudo: false
+
+language: node_js
+
+node_js:
+  - "node"
+
```
Comment (dinorhythms) [inline travis.yml]: can you add the install and script that runs like below?
```
install:
  - npm install

script:
  - npm test
  - npm run coverage

OR 

install:
  - npm install

script:
  - npm test

after_success:
 - npm run coverage
```

**Review by dinorhythms — CHANGES_REQUESTED** [review]

File: travis.yml (hunk 1 above)
Comment (jsbuddy) [reply travis.yml]: Done.. I'll make sure i remember next time

File: travis.yml (hunk 2 above)
Comment (jsbuddy) [reply travis.yml]: I just added the install and script commands.. doesn't seems to change anything.
still trying to figure out whats wrong.

Comment (jsbuddy) [reply travis.yml]: I fixed the issue @dinorhythms.. thanks

**Review by tejiri4 — APPROVED** [review]
LGTM

**Pull request merged**